lazy_tensor_core/test/test_lazy.py

```diff
@@ -1,13 +1,38 @@
 import torch
 
-from torch.testing._internal.common_utils import run_tests
+from torch.testing._internal.common_utils import run_tests, TestCase
 from torch.testing._internal.jit_utils import JitTestCase
+from torch.testing._internal.common_methods_invocations import op_db
+from torch.testing._internal.common_device_type import ops, instantiate_device_type_tests
 
 import lazy_tensor_core
-
 lazy_tensor_core._LAZYC._ltc_init_ts_backend()
-torch.manual_seed(42)
+import lazy_tensor_core.core.lazy_model as ltm
+import lazy_tensor_core.debug.metrics as metrics
+import itertools
+import yaml
+import os
+import pathlib
+
+def remove_suffixes(l):
+    return [x.split(".")[0] for x in l]
+
+def init_lists():
+    path_to_script = pathlib.Path(os.path.abspath(os.path.dirname(__file__)))
+    TS_NATIVE_FUNCTIONS_PATH = os.path.join(*path_to_script.parts[:-1], "ts_native_functions.yaml")
+    yaml_ts = yaml.load(open(TS_NATIVE_FUNCTIONS_PATH), yaml.Loader)
+    LAZY_OPS_LIST = set(remove_suffixes(itertools.chain(yaml_ts["full_codegen"], yaml_ts["supported"], yaml_ts["autograd"])))
```

Comment:
your code is simple enough i hesitate to say this, ~~but at least consider whether using the similar function in the codegen folder is better.~~

https://github.com/pytorch/pytorch/blob/master/tools/codegen/gen_lazy_tensor.py#L25

as i look at it, the existing codegen utils wouldn't be directly useful here, you'd have to still use more than one util to reconstruct all 3 lists...

Reply:
yeah, I looked at lazy_ir.py and gen_lazy_tensors.py and frankly it looked like an overkill to parse `ts_backend.yml` into NativeFunctions for the sole purpose of getting op names that LT supports O_O